src/types/responses/PalletStakingProgress.ts

```diff
@@ -10,9 +10,11 @@ export interface IPalletStakingProgress {
 	nextActiveEraEstimate?: AnyJson;
 	nextSessionEstimate: string | null;
 	unappliedSlashes: AnyJson[] | null;
-	electionStatus?: {
-		status: AnyJson;
-		toggleEstimate: string | null;
-	};
+	electionStatus?:
+		| {
+				status: AnyJson;
+				toggleEstimate: string | null;
+		  }
+		| string;
```

Comment:
dq: What do the `|`s do here?

Reply:
So here we have a Union Type, the first `|` denotes that the type has not yet been decided but it will guaranteed be one of the following below. And all following `|`'s are there to allow `electionStatus` to fulfill multiple types.